### PR TITLE
[Component] Chips

### PR DESCRIPTION
**Pull request by JLYamin**
## Linear Issue: [NUI-86](https://linear.app/novatics-ui/issue/NUI-86/chip)

Changes the chip component style to match the [Figma specifications](https://www.figma.com/file/kPAvItMtijRWkMzz8yPS1S/Design-Systemtics?node-id=1442%3A53207&t=7Zm0ow28sNGt5sle-1).

### CODE REVIEW

File: packages/styles/src/lib/Theme/components/MuiChip.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
+import { Components } from '@mui/material';
+import { palette as themePalette } from '../palette';
+import { themeTypography } from '../typography';
+import '../fonts.css';
+
+// This adds 50% transparency to hexadecimal colors
+const addTransparency = (color?: string): string => `${color}80`;
```
Comment (mgonc) [inline packages/styles/src/lib/Theme/components/MuiChip.ts:7]: can we make this function more customizable, passing the transparency and make it as a util inside of styles package?

File: packages/styles/src/lib/Theme/components/MuiChip.ts [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    case 'info':
+      return themePalette.support?.light;
+    default:
+      return themePalette.grayScale?.newMoon;
+  }
+};
+
+export const MuiChip: Components['MuiChip'] = {
+  styleOverrides: {
+    root: ({ ownerState }) => ({
```
Comment (mgonc) [inline packages/styles/src/lib/Theme/components/MuiChip.ts:43]: nice touch

File: packages/styles/src/lib/Theme/fonts.css [hunk 3]
```diff
@@ -0,0 +1 @@
+@import url('https://fonts.googleapis.com/css2?family=Work+Sans:wght@300;400;500;700&display=swap');
```
Comment (mgonc) [inline packages/styles/src/lib/Theme/fonts.css:1]: Thanks for add our font

File: packages/styles/src/lib/Theme/palette.ts [hunk 4]
```diff
@@ -82,6 +91,10 @@ export const palette: PaletteOptions = {
     dark: colors['mars--dark'],
     light: colors['mars--light'],
   },
+  support: {
+    main: colors['universe'],
```
Comment (mgonc) [inline packages/styles/src/lib/Theme/palette.ts:95]: do we have dark variant?

File: packages/chip/src/lib/chip.stories.mdx [hunk 5]
```diff
@@ -0,0 +1,349 @@
...
+            variant="outlined"
+          />
+        </div>
+      );
+    }}
+  </Story>
+</Canvas>
+
+```tsx
+<Chip label="Clickable Chip" clickable />
```
Comment (mgonc) [inline packages/chip/src/lib/chip.stories.mdx:79]: no need to add sample of code. When we use canvas, storybook provides a "show code" option
image

File: packages/styles/src/lib/Theme/components/MuiChip.ts [hunk 6]
```diff
@@ -0,0 +1,96 @@
+import { Components } from '@mui/material';
+import { palette as themePalette } from '../palette';
+import { themeTypography } from '../typography';
+import '../fonts.css';
```
Comment (mgonc) [inline packages/styles/src/lib/Theme/components/MuiChip.ts:4]: please move to createTheme file

Comment (mgonc) [inline packages/styles/src/lib/Theme/components/MuiChip.ts:4]: please move to createTheme file

**Review by mgonc — CHANGES_REQUESTED** [review]

File: packages/styles/src/lib/Theme/components/MuiChip.ts (hunk 1 above)
Comment (JLYamin) [reply packages/styles/src/lib/Theme/components/MuiChip.ts:7]: It's a little bit confusing to read but sure, no problem

**Review by mgonc — APPROVED** [review]

**Pull request merged**